### PR TITLE
Getting Started: Precompiled binaries

### PR DESCRIPTION
**Pull request by szczys**
* Summary page
* Thingy91 guide
* Update docusaurus (need for nested admonitions)

### CODE REVIEW

File: docs/getting-started/3-device-examples/1-precompiled-binaries/README.md [hunk 1]
```diff
@@ -4,11 +4,19 @@ sidebar_position: 1
...
-Golioth currently offers pre-compiled binaries for three different devices:
+The quickest way to try out Golioth is by using one of the test applications
+that we offer. These include firmware binaries that are ready to flash to your
+hardware. Once completed, you will be able to assign your own device credentials
+to get connected.
 
-* Nordic Thingy:91
-* Nordic nRF9160 DK
-* Nordic nRF7002 DK
+These test apps send and received data, listen for fleetwide settings,
```
Comment (sam-golioth) [inline docs/getting-started/3-device-examples/1-precompiled-binaries/README.md:12]:
```suggestion
These test apps send and receive data, listen for fleetwide settings,
```

File: docs/getting-started/3-device-examples/1-precompiled-binaries/_partials/2-register.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* Enter a Project Name of your choosing
+
+### Step 2: Device name
+
+* Enter a human-readable Device Name
+
+### Step 3: Device credentials
+
+* The Identity of this device is automatically populated from the device name with `-id` and `@project-name` appended
+* A Pre-Shared Key (PSK) is automatically generated. This is a password that will authenticate this device to the Golioth Cloud
```
Comment (sam-golioth) [inline docs/getting-started/3-device-examples/1-precompiled-binaries/_partials/2-register.md:46]:
```suggestion
* A Pre-Shared Key (PSK) is automatically generated. This is a password that will authenticate this device to the Golioth Cloud.
```

File: docs/getting-started/3-device-examples/1-precompiled-binaries/_partials/4-authorize-devices.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Key) are what your device needs to authenticate and connect to the Golioth
+Cloud. You can always return to this panel in the device details to retrieve
+these values.
+
+:::note
+When a device credential ID is automatically assigned it follows this
+format: GMT timestamp, the name of the device, and the name of the project. If
+you prefer shorter device IDs, you may want to choose the ID yourself. By
+convention, all device credential IDs will end with the `@` symbol followed by
+the project name.
```
Comment (sam-golioth) [inline docs/getting-started/3-device-examples/1-precompiled-binaries/_partials/4-authorize-devices.md:24]:
```suggestion
the project slug.
```

**Review by sam-golioth — APPROVED** [review]
Looks good! Just a few minor comments

File: docs/getting-started/3-device-examples/1-precompiled-binaries/_partials/2-register.md (hunk 2 above)
Comment (szczys) [reply docs/getting-started/3-device-examples/1-precompiled-binaries/_partials/2-register.md:46]: Ah, sorry for the extra document review here, mea culpa. These three numbered partials should not be included. I must have copied the whole directory from our developer-training repo when I only meant to copy the nrf-connect partial. I have removed the files and verified the the production build works without them.

**Pull request merged**